From: Ben Hutchings <ben@decadent.org.uk>
Date: Thu, 18 Feb 2016 23:37:19 +0000
Subject: [PATCH] Revert "perf tools, x86: Build perf on older user-space as
 well"
Forwarded: no

This reverts commit eae7a755ee81129370c8f555b0d5672e6673735d.
The empty unistd_{32,64}.h headers prevent building on sh, which also
now splits its unistd.h this way.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 tools/perf/perf-sys.h                   | 6 ------
 tools/perf/util/include/asm/unistd_32.h | 1 -
 tools/perf/util/include/asm/unistd_64.h | 1 -
 3 files changed, 8 deletions(-)
 delete mode 100644 tools/perf/util/include/asm/unistd_32.h
 delete mode 100644 tools/perf/util/include/asm/unistd_64.h

diff --git a/tools/perf/perf-sys.h b/tools/perf/perf-sys.h
index 83a25cef82fd..182a84eb5bd6 100644
--- a/tools/perf/perf-sys.h
+++ b/tools/perf/perf-sys.h
@@ -11,9 +11,6 @@
 #if defined(__i386__)
 #define cpu_relax()	asm volatile("rep; nop" ::: "memory");
 #define CPUINFO_PROC	{"model name"}
-#ifndef __NR_perf_event_open
-# define __NR_perf_event_open 336
-#endif
 #ifndef __NR_futex
 # define __NR_futex 240
 #endif
@@ -25,9 +22,6 @@
 #if defined(__x86_64__)
 #define cpu_relax()	asm volatile("rep; nop" ::: "memory");
 #define CPUINFO_PROC	{"model name"}
-#ifndef __NR_perf_event_open
-# define __NR_perf_event_open 298
-#endif
 #ifndef __NR_futex
 # define __NR_futex 202
 #endif
diff --git a/tools/perf/util/include/asm/unistd_32.h b/tools/perf/util/include/asm/unistd_32.h
deleted file mode 100644
index 8b137891791f..000000000000
--- a/tools/perf/util/include/asm/unistd_32.h
+++ /dev/null
@@ -1 +0,0 @@
-
diff --git a/tools/perf/util/include/asm/unistd_64.h b/tools/perf/util/include/asm/unistd_64.h
deleted file mode 100644
index 8b137891791f..000000000000
--- a/tools/perf/util/include/asm/unistd_64.h
+++ /dev/null
@@ -1 +0,0 @@
-
